components/menu-dropdown/menu-dropdown.jsx

```diff
@@ -370,6 +370,10 @@ const propTypes = {
 		PropTypes.string,
 		PropTypes.array,
 	]),
+	/**
+	 * Sets the variant to be used for the dropdown.
+	 */
+	variant: PropTypes.oneOf(['base', 'action']),
```

Comment:
Is this really necessary? Seems overkill to me. Could we use `className` prop to set `slds-dropdown_actions` on the dropdown menu instead?